### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by allencloud**
fix typos.
Signed-off-by: allencloud <allen.sun@daocloud.io>

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Netlink is also used to manage the routing table in the namespace.
 
 ## API
 
-Drivers are essentially an extension of libnetwork and provides the actual implementation for all of the LibNetwork APIs defined above. Hence there is an 1-1 correspondence for all the `Network` and `Endpoint` APIs, which includes :
+Drivers is essentially an extension of libnetwork and provides the actual implementation for all of the LibNetwork APIs defined above. Hence there is an 1-1 correspondence for all the `Network` and `Endpoint` APIs, which includes :
```
Comment (thaJeztah) [inline docs/design.md]: I think this should be "are" (because drivers is plural), but you can change "provides" to "provide" instead

**Comment by thaJeztah** [comment]
Thanks @allencloud, one comment, but LGTM otherwise (but I'm not a maintainer in this repo :smile:)

**Comment by allencloud** [comment]
@thaJeztah Thanks for your feedback.😄 Actually I am doing a deep learning of libnetwork.

File: docs/design.md (hunk 1 above)
Comment (allencloud) [inline docs/design.md]: updated that.

**Comment by thaJeztah** [comment]
thanks! LGTM

**Comment by mavenugo** [comment]
LGTM

**Pull request merged**